js/lib/scene.js

```diff
@@ -1,17 +1,16 @@
 define(['basic/entity', 'geo/v2', 'core/graphic', 'config/screen', 'config/config'],
 		function (Entity, V2, graphics, screen, config) {
-			function Scene() {
+		    function Scene() {
 				Entity.call(this, Zero(), new V2(screen.w, screen.h));
 				this.keyAware = [];
+			    this.bgColor = "#DDD";
 			}
 
 			Scene.prototype = new Entity();
 
 			Scene.prototype.onDraw = function (ctx) {
-				if (config.debug) {
-					ctx.fillStyle = "#dddddd";
-					ctx.fillRect(0, 0, this.size.x, this.size.y);
-				}
+				ctx.fillStyle = this.bgColor;
+				ctx.fillRect(0, 0, this.size.x, this.size.y);
```

Comment:
please surround by if statement and check if the color is set. otherwise we would have unnecessary draw calls for scenes that don't use this feature. You can put it in an else if for the existing bg check to ensure that not both functions are used at the same time.